### ISSUE TITLE
onDisconnect not firing

### ISSUE THREAD

**Issue by devcryp** [dparlevliet/node.bittrex.api#36]
First of all, thank you for sharing this module! I am experiencing an issue where onDisconnect is not triggered when I loose my internet connection (test by unplugging computer from the internet). Also when I turn on the verbose option there is no message notifying me about the loss of connection. 

Overwriting serviceHandlers.connectionLost or serviceHandlers.disconnected doesn't work for me either. Is there something I am doing wrong? 

`bittrex.options({
		
		'apikey' : 'key',
		'apisecret' : 'secret', 
		'inverse_callback_arguments': true,
		'verbose': true,
		'websockets': {
    		
    		onDisconnect: function() {
      			
      			console.log('Init: websocket disconnected not triggered');
    		},
    	},
	});`

Thank you! Joe

**Comment by dparlevliet** [comment]
Nope, you're not doing anything wrong. The ``onDisconnect`` is called inside the ``disconnected`` service handler by dafault so if that service handler isn't firing then that would be an issue with the third-party library. It is possible for the websocket controller to not know that a disconnection has occurred, pulling the plug might cause such an event might only happen after a lengthy timeout or if the timeout isn't handled in the library then it might not happen at all. I'm not sure, I can only speculate.

I can try look at this sometime soon - I'm in the middle of a heap of IRL stuff atm and wont be finished with that for 2 weeks. If this is a concern you need to fix fast then I'd suggest having a timer that checks your internet connectivity periodically.
